src/pages/SigningOfficialDataSubmitters.js

```diff
@@ -45,6 +46,7 @@ export default function SigningOfficialConsole() {
       div({style: {}, className: 'signing-official-tabs'}, [
         //NOTE: Links to this custodian table have been removed, we are retaining it with the intention of repurposing it for data submitters
         h(DataCustodianTable, {researchers, signingOfficial, unregisteredResearchers, isLoading}, []),
+        h(SigningOfficialDAAPopup, {}),
```

Comment:
the logic for determining acceptance status, showing, and accepting is all in the popup class - no logic needed outside to accomplish this, just need to mount the popup component.